packages/atomic/src/pages/layouts/search.html

```diff
@@ -0,0 +1,69 @@
+<!-- DO NOT TOUCH, THE PURPOSE OF THIS FILE IS FOR VISUAL COMPARISON USING CYPRESS -->
+<!DOCTYPE html>
+<html dir="ltr" lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0, minimum-scale=1.0, maximum-scale=5.0" />
+    <title>Coveo Atomic</title>
+
+    <script type="module" src="/build/atomic.esm.js"></script>
+    <script nomodule src="/build/atomic.js"></script>
+    <link rel="stylesheet" href="/themes/coveo.css" />
+    <script>
+      (async () => {
+        await customElements.whenDefined('atomic-search-interface');
+        const searchInterface = document.querySelector('atomic-search-interface');
+
+        await searchInterface.initialize({
+          accessToken: 'xx564559b1-0045-48e1-953c-3addd1ee4457',
+          organizationId: 'searchuisamples',
+          search: {
+            preprocessSearchResponseMiddleware: (response) => {
+              response.body.totalCountFiltered = 100;
+              const firstResult = response.body.results[0];
+              firstResult.title = 'First result';
+              firstResult.clickUri = 'https://docs.coveo.com/#' + Math.random();
```

Comment:
Why the math.random() ? 

Isn't this defeating the purpose of always having the exact same result on every run ? It does not seem to serve any purpose at all.

Reply:
When testing locally vs the CI, the link can sometimes be "visited" which is a slightly different style (purple text)

Reply:
I see ! 

Alright, can you add a comment to this effect ?